README.md

```diff
@@ -2,54 +2,76 @@
+
+Stager uses Scenes (see the [Concepts](#concepts) section below) that you define to generate small Flutter apps. To run the Stager apps included in the example, start by moving to the `example` directory and fetching the app's dependencies:
+
+```bash
+cd example
+flutter pub get
+```
+
+You can then run the indivdual Stager apps with the following commands:
+
+**Posts List**
+```bash
+flutter run -t lib/pages/posts_list/posts_list_page_scenes.stager_app.g.dart
+```
+
+**User Detail**
+```bash
+flutter run -t lib/pages/user_detail/user_detail_page_scenes.stager_app.g.dart
+```
 
-A Stager app for a ListView displaying forum-style posts:
+**Post Detail**
+```bash
+flutter run -t lib/pages/post_detail/post_detail_page_scenes.stager_app.g.dart
+```
 
-![example app demo](https://user-images.githubusercontent.com/581764/181614468-cbb89cbe-d16a-44bf-831e-47139ce3a7c0.gif)
+To get an idea of how these Scenes fit together, you can also run the main app by executing `flutter run` from the `example` directory.
 
 ## Concepts
 
 ### Scene
 
-The most important class in Stager is the Scene class. A Scene is a simple, self-contained unit of UI. Scenes make it easy to focus on a single widget or page to greatly increase development velocity by isolating them from the rest of your app and allowing fine control of dependencies.
-
-A Scene has three parts:
+A Scene is a simple, self-contained unit of UI, and is the most important idea in Stager. Scenes make it easy to focus on a single widget or page to greatly increase development velocity by isolating them from the rest of your app and allowing fine control of dependencies.
 
-#### `title`
+To create your own Scene, simply create a `Scene` subclass and implement `title`, the name of your Scene, and `build()`, which constructs body of the Scene.
 
-The name of the Scene.
+You can also override the following methods and properties:
 
 #### `setUp`
 
 A function that is called once before the Scene is displayed. This will generally be where you configure your widget's dependencies.
 
-#### `build`
-
-A function that constructs your widget.
+#### `environmentControlBuilders`
 
-### StagerApp
+An optional list of `WidgetBuilder`-like functions that allow you to add custom Widgets to the Stager control panel. These are useful if you want to manipulate things specific to your app, including:
 
-A StagerApp displays a list of Scenes, allow the user to select from all available Scenes. Because Scenes can contain their own Navigators, the StagerApp overlays a back button on top of the Scenes.
+- Data displayed by your Widget
+- Properties on mocked dependenices
+- Feature flags
 
-## Demo
+You can return any arbitrary Widget from these functions.
 
-See the example directory for a demo that highlights some of the useful things Stager allows you to do, including:
+### StagerApp
```

Comment:
You buried the lode a bit further day, but maybe just clarify right out the gate that you don't need to manually create this?